### ISSUE TITLE
fix(Strategy): cancel exit orders

### ISSUE THREAD

**Issue by DevNest360** [DevNest360/where#2]
in case i want to cancel both of the exit orders without setting new exit orders I would like to do something like this:

```
self.stop_loss = []
self.take_profit = [] 
```
but it will raise an error of `'stop-loss and take-profit should not be exactly the same. Just use either one of them and it will do.'`
